core/esmf-aspect-meta-model-java/src/main/java/org/eclipse/esmf/aspectmodel/serializer/RdfModelCreatorVisitor.java

```diff
@@ -646,10 +647,14 @@ public ElementModel visitProperty( final Property property, final ModelElement c
       model.add( serializeDescriptions( resource, property ) );
 
       property.getExampleValue().ifPresent( exampleValue -> {
-         final ElementModel exampleValueElementModel = exampleValue.accept( this, property );
-         model.add( exampleValueElementModel.model() );
-         exampleValueElementModel.focusElement().ifPresent( exampleValueNode ->
-               model.add( resource, SammNs.SAMM.exampleValue(), exampleValueNode ) );
+         if ( exampleValue instanceof Value value ) {
```

Comment:
If `property.getExampleValue()` always returns a `ScalarValue` as before, this check can be removed